/camera1/camera1_node:
  ros__parameters:
    frame_logged: 50
    URI: "http://192.168.1.172:8000/video_feed"
    quality: 80
    compression_format: ".jpg"

/camera2/camera2_node:
  ros__parameters:
    frame_logged: 50
    URI: "http://192.168.1.172:8001/video_feed"
    quality: 80
    compression_format: ".jpg"


# Note: if a different compression format is choose, 'encode_param' in CameraNode must be modified